src/components/App.js

```diff
@@ -1,14 +1,35 @@
 /* eslint-disable linebreak-style */
-import React from 'react';
+import React, { Component } from 'react';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
-import calculate from '../logic/calculate';
-
-const App = () => (
-  <>
-    <Display />
-    <ButtonPanel onClick={calculate} />
-  </>
-);
+import Calculate from '../logic/calculate';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
```

Comment:
**[Optional]**
You could use arrow function in the function definition of `handleClick` to save your self from binding it in the constructor.